sign/eddilithium3/example_test.go

```diff
@@ -0,0 +1,45 @@
+package eddilithium3_test
+
+import (
+	"fmt"
+
+	"github.com/cloudflare/circl/sign/eddilithium3"
+)
+
+func Example() {
```

Comment:
Why is this not a "regular" unit test? (Passing in `t *testing.T`, and having a proper name.)

Reply:
This is a package wide testable example that appears after the description in the godoc. See https://blog.golang.org/examples

Reply:
Got it. Do we have actual unit tests?

Reply:
Yes, `TestSignThenVerifyAndPkSkPacking` and `TestPublicFromPrivate` in the `_test.go`.